invenio_rdm_records/records/jsonschemas/records/definitions-v2.0.0.json

```
@@ -384,5 +384,15 @@
         "$ref": "local://definitions-v1.0.0.json#/identifier"
       }
     }
+  },
+  "deletion_policy": {
+    "type": "object",
+    "additionalProperties": false,
+    "description": "Deletion policy that was applied to this record.",
```

```suggestion
    "description": "Deletion policy that applied to this record.",
```

I think it should be like this (same with below)